Failure on ubuntu-latest (jammy) for armhf

Failure message is at the bottom.   Relevant bits of my workflow file:
```
name: Build
on:
  push:
    branches:
      - develop
  pull_request:
     types: [opened, synchronize]
jobs:
  build:
    runs-on: ${{ matrix.config.os }}
    strategy:
      matrix:
        config:
          - { os: ubuntu-latest, arch: x64 }
    steps:
    - uses: cyberjunk/gha-ubuntu-cross@v3
      with:
        arch: armhf

```

end of output:
```
Ign:30 http://security.ubuntu.com/ubuntu jammy-security/restricted armhf Packages
Ign:33 http://security.ubuntu.com/ubuntu jammy-security/universe armhf Packages
Ign:37 http://security.ubuntu.com/ubuntu jammy-security/multiverse armhf Packages
Get:54 http://ports.ubuntu.com/ jammy-updates/main Translation-en [214 kB]
Get:55 http://ports.ubuntu.com/ jammy-updates/main amd64 c-n-f Metadata [15.6 kB]
Get:56 http://ports.ubuntu.com/ jammy-updates/restricted armhf Packages [10.1 kB]
Get:57 http://ports.ubuntu.com/ jammy-updates/restricted Translation-en [114 kB]
Get:58 http://ports.ubuntu.com/ jammy-updates/restricted amd64 c-n-f Metadata [536 B]
Get:59 http://ports.ubuntu.com/ jammy-updates/universe armhf Packages [714 kB]
Get:60 https://packages.microsoft.com/ubuntu/22.04/prod jammy InRelease [3611 B]
Get:61 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main armhf Packages [9004 B]
Get:62 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main arm64 Packages [18.1 kB]
Get:63 https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 Packages [87.5 kB]
Get:64 http://ports.ubuntu.com/ jammy-updates/universe Translation-en [212 kB]
Get:65 http://ports.ubuntu.com/ jammy-updates/universe amd64 c-n-f Metadata [21.6 kB]
Get:66 http://ports.ubuntu.com/ jammy-updates/multiverse armhf Packages [3164 B]
Get:67 http://ports.ubuntu.com/ jammy-updates/multiverse Translation-en [9768 B]
Get:68 http://ports.ubuntu.com/ jammy-updates/multiverse amd64 c-n-f Metadata [476 B]
Get:69 http://ports.ubuntu.com/ jammy-backports/main armhf Packages [40.5 kB]
Get:70 http://ports.ubuntu.com/ jammy-backports/main Translation-en [10.2 kB]
Get:71 http://ports.ubuntu.com/ jammy-backports/main amd64 c-n-f Metadata [388 B]
Get:72 http://ports.ubuntu.com/ jammy-backports/restricted amd64 c-n-f Metadata [116 B]
Get:73 http://ports.ubuntu.com/ jammy-backports/universe armhf Packages [20.6 kB]
Get:74 http://ports.ubuntu.com/ jammy-backports/universe Translation-en [15.4 kB]
Get:75 http://ports.ubuntu.com/ jammy-backports/universe amd64 c-n-f Metadata [576 B]
Get:76 http://ports.ubuntu.com/ jammy-backports/multiverse amd64 c-n-f Metadata [116 B]
Fetched 30.4 MB in 7s (4110 kB/s)
Reading package lists...
E: Failed to fetch http://security.ubuntu.com/ubuntu/dists/jammy/main/binary-armhf/Packages  404  Not Found [IP: 52.154.174.208 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/dists/jammy-updates/main/binary-armhf/Packages  404  Not Found [IP: 52.154.174.208 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/dists/jammy-backports/main/binary-armhf/Packages  404  Not Found [IP: 52.154.174.208 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/dists/jammy-security/main/binary-armhf/Packages  404  Not Found [IP: 52.154.174.208 80]
E: Some index files failed to download. They have been ignored, or old ones used instead.
Error: Process completed with exit code 100.
```

--- a/action.yml
+++ b/action.yml
@@ -27,8 +27,10 @@ runs:
             deb [arch=$ARCH] http://ports.ubuntu.com/ $CODENAME-backports main restricted universe multiverse \n' > $APTFILE"; fi
         if [ $ARCH == i386 ]; then \
           sudo sed -i 's/deb http/deb [arch=amd64,i386] http/g' /etc/apt/sources.list; \
+          sudo sed -i 's/deb mirror/deb [arch=amd64,i386] mirror/g' /etc/apt/sources.list; \
         else \
           sudo sed -i 's/deb http/deb [arch=amd64] http/g' /etc/apt/sources.list; \
+          sudo sed -i 's/deb mirror/deb [arch=amd64] mirror/g' /etc/apt/sources.list; \
         fi
         sudo apt-get update
         sudo apt-get install \

